src/ir/ty.rs

```diff
@@ -365,6 +353,16 @@ impl IsOpaque for Type {
     }
 }
 
+/// Whether this type has a vtable.
+impl HasVtable for Type {
+    // Note that we need the Type's Item to answer this question.
+    type Extra = ItemId;
+
+    fn has_vtable(&self, ctx: &BindgenContext, itemid: &ItemId) -> bool {
+        ctx.lookup_item_id_has_vtable(itemid)
+    }
```

Comment:
I'm on the edge... do you think this trait and these impls are carrying their weight? I guess I like the `Item` and `ItemId` impls, but this one seems pretty shaky to me...